### PR TITLE
Various comment edits (typos, consistency).

### PR DESCRIPTION
**Pull request by benjub**
Various minor edits. Nothing systematic, but rather opportunistic fixes. Aiming for consistency.

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -548522,34 +548532,34 @@ axioms listed above ( ~ axc11 appears slightly stronger since
       ( wnf wn wi wal wex nfv bj-nexdt ax-mp ) ACDABEFCGABCHEFFACIABCJK $.
   $}
 
-  $( Adding a second quantifier is a tranparent operation, ( ` A. E. ` case).
-     (Contributed by BJ, 20-Oct-2019.) $)
+  $( Adding a second quantifier over the same variable is a tranparent
```
Comment (icecream17) [inline set.mm:548535]:
```suggestion
  $( Adding a second quantifier over the same variable is a transparent
```

x4
bj-{al,ex}exbiex
bj-{al,ex}albial

File: set.mm [hunk 2]
```diff
@@ -679284,7 +679293,7 @@ base set if and only if the neighborhoods (convergents) of every point
                     u - v ) ) ) = ( 2 x. ( ( F ` u ) x. ( G ` v ) ) ) ) $.
     imo72b2.6 $e |- ( ph -> A. y e. RR ( abs ` ( F ` y ) ) <_ 1 ) $.
     imo72b2.7 $e |- ( ph -> E. x e. RR ( F ` x ) =/= 0 ) $.
-    $( IMO 1972 B2.  (14th International Mathemahics Olympiad in Poland,
+    $( IMO 1972 B2.  (14th International Mathemathics Olympiad in Poland,
```
Comment (icecream17) [inline set.mm:679296]:
```suggestion
    $( IMO 1972 B2.  (14th International Mathematical Olympiad in Poland,
```
searching it up online, this is the actual acronym for the IMO, whose 14th occurrence did indeed happen in Poland: https://en.wikipedia.org/wiki/List_of_International_Mathematical_Olympiads

**Review by icecream17 — APPROVED** [review]
Not sure about the `If ..., [+then] ...` but it's overall all good.

File: set.mm [hunk 3]
```diff
@@ -20804,10 +20805,11 @@ theorem as an axiom of set theory (Axiom 0 of [Kunen] p. 10).  In the
...
     $( Existential introduction, using implicit substitution.  Compare Lemma 14
        of [Tarski] p. 70.  Usage of this theorem is discouraged because it
-       depends on ~ ax-13 .  Check out ~ spimew for a weaker version requiring
-       fewer axioms.  (Contributed by NM, 7-Aug-1994.)  (Revised by Mario
-       Carneiro, 3-Oct-2016.)  (Proof shortened by Wolf Lammen, 6-Mar-2018.)
-       (New usage is discouraged.) $)
+       depends on ~ ax-13 .  See ~ spimew and ~ spimevw for weaker versions
+       requiring fewer axioms.  (Contributed by NM, 7-Aug-1994.)  (Revised by
+       Mario Carneiro, 3-Oct-2016.)  (Proof shortened by Wolf Lammen,
+       6-Mar-2018.)  Usage of this theorem is discouraged because it depends on
```
Comment (GinoGiotto) [inline set.mm:20811]: `Usage of this theorem is discouraged because it depends on ~ ax-13` is written two times.

**Comment by benjub** [comment]
Thank you for the reviews. About the "then" question above, and a few punctuation choices, I recommend the very widely recommended short article:

Paul Halmos, How to Write Mathematics, available at https://sites.math.washington.edu/~lind/Resources/Halmos.pdf

It has one section about punctuation (and the other sections are even more important).

**Review by avekens — APPROVED** [review]

**Pull request merged**